nixos/modules/virtualisation/waydroid.nix

```diff
@@ -8,7 +8,6 @@
 let
   cfg = config.virtualisation.waydroid;
   kCfg = config.lib.kernelConfig;
-  kernelPackages = config.boot.kernelPackages;
```

Comment:
Is this necessary?

Reply:
It doesn't seem to be used in the file. `config.boot.kernelPackages` is disectly used in an assertion